### ISSUE TITLE
[stable/rabbitmq] ulimit doesn't work on AWS

### ISSUE THREAD

**Issue by kytart** [helm/charts#14020]
I'm running my Kubernetes cluster on AWS EC2 and all the nodes run on top of Amazon Linux 2. 

RabbitMQ stateful set has a explicit command that contains this
```
ulimit -n "${RABBITMQ_ULIMIT_NOFILES}"
```

Unfortunatelly, AWS doesn't allow that. When I run the container, it fails on this line with error message:
> error setting limit: Operation not permitted

I found some related answers:
https://stackoverflow.com/questions/11342167/how-to-increase-ulimit-on-amazon-ec2-instance
https://gist.github.com/AlexVKO/1b6c7df44bbe152ca25dfe7e30b21f3f

Basically, since this doesn't work and the container fails because of this, I'd like to have an option to skip this line completely and manage the max file descriptor limit externally.

I have a PR ready that adds this feature. I'll post it right after.

**Comment by denis111** [comment]
Yes, also, when you create AWS node you can put this into your userdata when you create ec2 instance
```
#!/bin/bash
sed -i -e 's/1024:4096/65536:65536/g' /etc/sysconfig/docker
sed -i -e 's/2048/65536/g' /etc/docker/daemon.json
sed -i -e 's/8192/65536/g' /etc/docker/daemon.json
```

**Comment by javsalgar** [comment]
Hi,

Thanks for letting us know. I think it makes sense to add an option to skip it as not all the target platforms may support it. Please let us know when the PR is ready to be reviewed.

**Comment by kytart** [comment]
@javsalgar great, thanks. The PR is ready #14022
